x-pack/plugins/triggers_actions_ui/public/application/sections/alert_form/alert_notify_when.tsx

```diff
@@ -29,13 +29,13 @@ const DEFAULT_NOTIFY_WHEN_VALUE: AlertNotifyWhenType = 'onActionGroupChange';
 const NOTIFY_WHEN_OPTIONS: Array<EuiSuperSelectOption<AlertNotifyWhenType>> = [
   {
     value: 'onActionGroupChange',
-    inputDisplay: 'Run only on status change',
+    inputDisplay: 'Only on status change',
```

Comment:
**optional nit:** Not related to this PR but it would be nice to have all the `inputDisplay` values go through i18n.

Reply:
Updated!